src/components/enterprise-user-subsidy/tests/UserSubsidy.test.jsx

```diff
@@ -66,9 +69,37 @@ describe('without subscription plan', () => {
   });
 });
 
-describe('with subscription plan', () => {
+describe('with inactive subscription plan', () => {
```

Comment:
nit: `inactive` here makes me think that the field `is_active` is `False`

Reply:
`is_active` _should_ be false here but it's not enforced so feels a tiny bit weird, but small potatoes

Reply:
I changed up the language to avoid using inactive/active in the test names to reduce that confusion. 

By the time we get to `<UserSubsidy>`, we already know that `is_active` is enabled for the subscription plan since that API call is made higher up in the component tree, and filters for `is_active`.